tripleblindmarket/private-kit
# [SAF-825] Fix exposure notification display

#### Description:

This PR fixes an issue with exposure notification screen showing incorrect message on selected date until user clicks away and back again

#### How to test:

- go to exposure notification screen

- select a day

- do something to trigger exposure notification ( subscribe / unsubscribe to HA or toggle exposure feature flag)

- return to exposure notification screen

- if the data for selected day changed, notification should also update
